prow/plugins/releasenote/releasenote.go

```diff
@@ -57,11 +58,13 @@ var (
 		releaseNoteActionRequired,
 		releaseNoteLabelNeeded,
 		releaseNote,
+		releaseNoteFeature,
 	}
 
 	releaseNoteRe               = regexp.MustCompile(`(?mi)^/release-note\s*$`)
 	releaseNoteNoneRe           = regexp.MustCompile(`(?mi)^/release-note-none\s*$`)
 	releaseNoteActionRequiredRe = regexp.MustCompile(`(?mi)^/release-note-action-required\s*$`)
+	releaseNoteFeatureRe        = regexp.MustCompile(`(?mi)^/release-note-feature\s*$`)
```

Comment:
Please don't add a new command. We are trying to deprecate these commands in favor of using the release note block in the PR body as the source of truth. Otherwise a user can comment with something like `/release-note` without actually providing release note text. The only command that we still allow (and which still makes sense) is the `/release-note-none` command because for this case we don't expect the release note block to have any important content.

It would probably be best to handle `release-note-feature` similarly to the the way we handle `release-note-action-required` (look for a key word/phrase in the release note block).